presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -88,12 +101,23 @@ Property Name                                        Description
 
 ==================================================== ============================================================
 
+Hadoop catalog
+^^^^^^^^^^^^^^
+
+Iceberg connector supports Hadoop as Catalog
+
 .. code-block:: none
 
     connector.name=iceberg
-    iceberg.catalog.type=nessie
-    iceberg.catalog.warehouse=/tmp
-    iceberg.nessie.uri=https://localhost:19120/api/v1
+    hive.metastore.uri=hostname:port
+    iceberg.catalog.type=hadoop
+
+.. note::
+
+    To use Hadoop as Catalog with Iceberg table, catalog requires the ``hive.metastore.uri`` property
+    added even though it doesn't rely on metastore for metadata when using Hadoop Catalog. You
```

Comment:
Suggest 

To use Hadoop as catalog with Iceberg table requires the ``hive.metastore.uri`` property, even though it doesn't rely on metastore for metadata when using Hadoop Catalog.